src/main/java/trello/ui/pages/BoardPage.java

```diff
@@ -26,16 +26,16 @@
 public class BoardPage extends BasePage {
 
     @FindBy(css = ".placeholder")
-    private WebElement newListBtn;
+    private WebElement newListButton;
 
     @FindBy(name = "name")
-    private WebElement newListNameTxtBox;
+    private WebElement newListNameField;
 
     @FindBy(css = ".primary")
```

Comment:
Replace css by className for find WebElement please.

Reply:
done.